doc/source/getting_started/compatibility.rst

```diff
@@ -23,7 +23,7 @@ version.
 
 As new features are developed, every attempt is made to ensure backward
 compatibility from the client to the server. Backward compatibility is generally ensured for
-the 4 latest Ansys versions. For example, ``ansys-dpf-core`` module with 0.8.0 version has been
+the four latest Ansys versions. For example, the ``ansys-dpf-core`` module 0.8.0 version has been
 developed for Ansys 2023 R2 pre1 release, for 2023 R2 Ansys version. It is compatible with
```

Comment:
@Jenna--do you know what is meant by "pre1" in this sentence? Is it necessary. Does it date the document is someway unnecessarily?

Reply:
I know they do both major releases that correspond with the Ansys release versions, along with internal pre-releases, so that is what pre-1 means. I will let one of the DPF reviewers comment here to see if it is actually needed or not.

Reply:
The `pre1` marker is necessary as part of the version.
If you want something cleaner you could write `Ansys 2023 R2' first pre-release` but i feel like this is harder to link to `2023.2.pre1` which is the version tag used.

Reply:
Actually the sentence is weird as it mentions two Ansys versions. It should only mention the `2023 R2 Ansys version`